supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/RuntimeOutputMapping.scala

```diff
@@ -0,0 +1,32 @@
+package cromwell.backend.google.pipelines.v2alpha1
+
+import common.util.StringUtil._
+import cromwell.backend.google.pipelines.common.io.PipelinesApiWorkingDisk
+import cromwell.core.path.Path
+
+object RuntimeOutputMapping {
+
+  /**
+    * List of prefixes to be stripped away from runtime output paths before
+    * appending them to the cloud call root to generate the delocalization path.
+    * The goal is to reduce unnecessary long paths which keep repeating the workflow root
+    * as the workflow progresses
+    * 
+    * For instance:
+    * 
+    * file:////cromwell_root/bucket/workflow_name/6d777414-5ee7-4c60-8b9e-a02ec44c398e/call-A/file.txt
```

Comment:
Is `file:////cromwell_root/` really a prefix? I'd guess it would only be three slashes, two after the scheme plus one before the root.

Reply:
Yeah good catch